frame/contracts/src/storage/meter.rs

```diff
@@ -335,14 +346,13 @@ where
 		self.total_deposit.available(&self.limit)
 	}
 
-	/// True if the contract is alive.
-	fn is_alive(&self) -> bool {
-		matches!(self.own_contribution, Contribution::Alive(_))
-	}
```

Comment:
I would keep it around seems easier to read than in lining `matches!(self.contract_state(), ContractState::Alive)` in a few places

Reply:
could be, given the fact that we have `ContractState::Alive` and `Contribution::Alive()` the inline seems more meaningful... but I don't have a strong opinion here 
@agryaznov wdyt?